### PR TITLE
Components: Fix cursor for non-clickable LinkHeaders

### PR DESCRIPTION
**Pull request by peterhass**

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "2.9.0",
+  "version": "2.9.1",
```
Comment (atomiomi) [inline packages/components/package.json:14]: not updated in lock file

File: packages/components/package-lock.json [hunk 2]
```diff
@@ -13,12 +13,6 @@
         "regenerator-runtime": "^0.13.4"
       }
     },
-    "@goodhood/icons": {
```
Comment (atomiomi) [inline packages/components/package-lock.json:16]: why remove oodhood/icons?

File: packages/components/package.json (hunk 1 above)
Comment (peterhass) [reply packages/components/package.json:14]: we never update the lockfile anyway, it's super annoying we have it anyway. you have to install from root package

File: packages/components/package-lock.json (hunk 2 above)
Comment (peterhass) [reply packages/components/package-lock.json:16]: i assume because we moved it to root package some time ago

**Review by suprMax — APPROVED** [review]

**Pull request merged**